config/main.py

```diff
@@ -1694,8 +1696,8 @@ def warm_restart_bgp_timer(ctx, seconds):
 @click.pass_context
 def warm_restart_teamsyncd_timer(ctx, seconds):
     db = ctx.obj['db']
-    if seconds not in range(1,3600):
-        ctx.fail("teamsyncd warm restart timer must be in range 1-3600")
+    if seconds not in range(1,3600) and seconds != MAXIMUM_WARMRESTART_TIMER_VALUE:
+        ctx.fail("teamsyncd warm restart timer must be in range 1-3600 or value 9999")
```

Comment:
It is unclear to the user what the value `9999` means/does. More information is needed in the help and error messages, as well as related documentation in the Command Reference document.

Reply:
Is it possible to have word "disabled" or something like this for the utility?
9999 it is a magic constant,

Reply:
UPDATED, change the python click logic a little bit to accept both numbers and 'disable', print the exact same error msg via ctx.fail()